mix.exs

```diff
@@ -16,21 +22,41 @@ defmodule ColonelKurtz.MixProject do
     ]
   end
 
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
   # Run "mix help compile.app" to learn about applications.
   def application do
     [
       extra_applications: [:logger]
     ]
   end
 
+  def applications(:test), do: [:phoenix]
```

Comment:
This is a bit of a hack (same for the `compilers` specified below). We should probably spin up a proper phoenix app for testing with. (Does that go in `examples/` or in a subfolder of `test/`? I'm not sure.)

Reply:
That matches my intuition as well. I think just generating a phoenix app under the `test/` folder for testing makes sense but is that too heavy-handed of an approach? It might be unavoidable.

Reply:
Yeah, I was struggling with this yesterday. I looked at how [liveview does testing](https://github.com/phoenixframework/phoenix_live_view/tree/master/test/support). They don't have a full app, just the bits needed for their tests. We could do something similar to integration test the rendering code, and maybe test the ecto stuff separately. I'll look into it.